src/components/cylc/Mutation.vue

```diff
@@ -216,31 +262,76 @@ export default {
     },
 
     /* Execute the GraphQL mutation */
-    submit () {
+    async submit () {
       this.submitting = true
-      this.$refs.form.submit().then(response => {
+      this.$refs.form.submit().then(async response => {
         this.submitting = false
         if (response.status === mutationStatus.SUCCEEDED) {
-          // Close the form on success
-          this.close()
-          this.$emit('success')
+          if (this.isView) {
+            // form is open in a tab -> provide an alert to let the user know
+            // the command succeeded
+            await store.dispatch(
+              'setAlert',
+              new Alert('Command succeeded', 'green')
+            )
+          } else {
+            // form is open in a dialogue -> close the form on success
+            this.close()
+            this.$emit('success')
+          }
         } else if (response.status === mutationStatus.WARN) {
           this.warningMsg = response.message
         }
         // else if error, an alert is generated by AOTF
       })
     },
+
+    openInTab () {
+      // Navigate to the corresponding workflow then open the log view
+      // (no nav occurs if already on the correct workflow page)
+      this.$router.push({
+        name: 'Workspace',
+        params: {
+          workflowName: this.cylcObject.tokens.workflow
+        }
+      }).then(() => {
```

Comment:
The same thing we do in `Menu.vue` for the benefit of the standalone views (otherwise the open in new tab button would do nothing in these contexts).